doc/base/classes.xml

```diff
@@ -20609,6 +20609,40 @@
 				Set the [Mesh] resource for the instance.
 			</description>
 		</method>
+		<method name="set_morph_track_value"  >
```

Comment:
There's something weird with the indentation and tags (e.g. two stray spaces before the `>` here).

You should probably use doctool to let it generate the template automatically: `godot -doctool doc/base/classes.xml`.